lib/utils/yamlAstParser.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+const yaml = require('yaml-ast-parser');
+const fs = require('fs');
+const _ = require('lodash');
+const os = require('os');
+
+const astObject = {};
```

Comment:
@horike37 putting this here is a very dangerous pattern. This will immediately break if the util methods are used more than once in the same execution lifecycle or if there are multiple run in parallel since all calls are sharing the same object in memory. Instead the AST should be passed as a function parameter.